src/client/clientmap.h

```diff
@@ -143,6 +143,7 @@ class ClientMap : public Map, public scene::ISceneNode
 protected:
 	void reportMetrics(u64 save_time_us, u32 saved_blocks, u32 all_blocks) override;
 private:
+	bool isMeshOccluded(MapBlock *mesh_block, u16 mesh_size, v3s16 cam_pos_nodes);
```

Comment:
one more sorry: can this be made static?

Reply:
Done.